docs/data/data-grid/api-object/api-object.md

```diff
@@ -62,7 +62,7 @@ const CustomDataGrid = (props) => {
 ```
 
 :::warning
-The API object will be populated by the various plugins of the grid during the first render of the component.
+Various plugins of the grid populate the API object during the first render of the component.
```

Comment:
This is a good example of a grammatical "edge case" where the passive voice makes sense—as I understand in context, we want to emphasize "the API object." I would just replace the future tense "will be" with "is."

```suggestion
The API object is populated by the various plugins of the grid during the first render of the component
```